server/db/schema.js

```diff
@@ -2,12 +2,11 @@
  * Define db schema rules including collection names and indexes
  */
 
-const collections = {
+const databaseCollections = {
```

Comment:
Might be a good idea to add comments here explaining what each of these are - it's not immediately apparent from the names.